src/hotspot/share/gc/serial/serialHeap.cpp

```diff
@@ -112,3 +114,14 @@ void SerialHeap::safepoint_synchronize_end() {
     SuspendibleThreadSet::desynchronize();
   }
 }
+
+HeapWord* SerialHeap::allocate_loaded_archive_space(size_t word_size) {
+  MutexLocker ml(Heap_lock);
+  HeapWord* result = old_gen()->allocate(word_size, /* is_tlab = */ false);
+  return result;
```

Comment:
```suggestion
  return old_gen()->allocate(word_size, false /* is_tlab */);
```
Tighten the code...; GC code (see similar examples in the file) also adds the comment for bool parameters after that parameter.